transit/script/utils.py

```diff
@@ -0,0 +1,166 @@
+from pathlib import Path
+
+import geopandas as gpd
+import pandas as pd
+
+time_periods = ["EA", "AM", "MD", "PM", "EV"]
+
+def transit_assignment_filepaths(model_run_dir, time_periods):
+    return {t: Path(model_run_dir) / f"SFALLMSA{t}.DBF" for t in time_periods}
```

Comment:
ignore if irrelevant. do we need this dictionary to be ordered? Or random ordering is fine.
If we do need it to be ordered, then switch to using an OrderedDict https://docs.python.org/3/library/collections.html#collections.OrderedDict (and you probably have to build it via a list of tuples: `OrderedDict([(t, Path...) for t in time_periods])`)